app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,66 @@
+class AnswersController < ApplicationController
+
+  def create
+    @question = Question.find_by(id: params[:question_id])
+    @answer = Answer.new(answer_params)
+    if @answer.save
+      flash.notice = "Answer Created"
+      redirect_to question_path(@question)
+    else
+      redirect_to question_path(@question)
+    end
+  end
+
+  def edit
+    @question = Question.find_by(id: params[:question_id])
+    @answer = Answer.find_by(id: params[:id])
+  end
+
+  def update
+    @question = Question.find_by(id: params[:question_id])
+    @answer = Answer.find_by(id: params[:id])
+    if @answer.update(answer_params)
+      flash.notice = "Answer Updated"
+      redirect_to question_path(@question)
+    else
+      render :edit
+    end
+  end
+
+  def mark_best_answer
+    answer = Answer.find_by(id: params[:id])
+    if current_user == answer.question.creator
+      if answer_params[:is_best_answer] == "1"
```

Comment:
You could do this in one line without the if/else using `answer.mark_as_best_answer(answer_params[:is_best_answer] == "1")`